### PR TITLE
Antonclk clock 0.05 show weeknumber *bugfix*

### PR DESCRIPTION
**Pull request by DDDanny**
I fixed the typo and format of the new option in readme.md
Furthermore I removed my testdata "weekday name" hardcoded to "thursday" - would be a mess if it is always thursday ;)
and tagged the string "week" that it has to be added in locale-app

### CODE REVIEW

File: apps/antonclk/app.js [hunk 1]
```diff
@@ -188,9 +188,8 @@ function draw() {
     g.drawString(dateStr, x, y);
     if (weekDay || calWeek) {
       var dowwumStr = require("locale").dow(date);
-      dowwumStr = "thursday";
       if (calWeek)
-        dowwumStr = (weekDay ? dowwumStr.substr(0,Math.min(dowwumStr.length,6)) + (dowwumStr.length>=6 ? "." : "") : "week ") + "#" + ISO8601calWeek(date);
+        dowwumStr = (weekDay ? dowwumStr.substr(0,Math.min(dowwumStr.length,6)) + (dowwumStr.length>=6 ? "." : "") : "week ") + "#" + ISO8601calWeek(date); //TODO: locale for "week"
```
Comment (BartS23) [inline apps/antonclk/app.js:192]: Maybe it is better that you use the default short weekday names require("locale").dow(date**, 1**) if you have no space.
```
var dowwumStr;
if  (calWeek && weekDay) {
  dowwumStr = require("locale").dow(date, 1) + " #" + ISO8601calWeek(date);
} else if (calWeek) {
  dowwumStr = "week #" + ISO8601calWeek(date);
} else {
  dowwumStr = require("locale").dow(date);
}
```

**Comment by DDDanny** [comment]
Fixes #1259

File: apps/antonclk/app.js (hunk 1 above)
Comment (DDDanny) [reply apps/antonclk/app.js:192]: Geat Idea!  I'll consider on next patch

**Comment by gfwilliams** [comment]
Thanks - sorry - missed this!

**Pull request merged**